bigtop-packages/src/charm/zeppelin/layer-zeppelin/LICENSE

```diff
@@ -0,0 +1,177 @@
+
```

Comment:
You don't need to include LICENSE file to each charm - it is already done at the top of the project.